README.md

```diff
@@ -1 +1,3 @@
-# plasma
\ No newline at end of file
+# Plasma
+
+Plasma is a visual pattern generator script written in Lua. Created by [tehn](https://github.com/tehn), this repository is designed to provide an easy-to-use way to generate dynamic plasma patterns.
```

Comment:
Emphasized the purpose of the repository by including a short, non-comprehensive description to inform users what this repository's purpose is.